data-structures-linked-lists/prepend.js

```diff
@@ -0,0 +1,10 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported prepend */
+
+function prepend(list, value) {
+  const newlist = new LinkedList(value);
+  if (newlist.next === null) {
+    newlist.next = list;
+  }
```

Comment:
You don't really need to check if the node you _just made_ only has one value, since you already know the answer.